spec/changelog.yml

```diff
@@ -21,9 +21,6 @@
     - description: Add support for configuration links to kibana manifest.
       type: enhancement
       link: https://github.com/elastic/package-spec/pull/834
-    - description: Expand support to internal and external configuration_links defined in kibana manifest.
-      type: enhancement
-      link: https://github.com/elastic/package-spec/pull/844
```

Comment:
Removing this changelog entry as is a follow up for a feature released in the same version of the spec.